rules/REQUEST-930-APPLICATION-ATTACK-LFI.conf

```diff
@@ -87,14 +87,16 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
     tag:'language-multi',\
     tag:'platform-multi',\
     tag:'attack-lfi',\
-    tag:'paranoia-level/1',\
+    tag:'paranoia-level/%{tx.paranoia_level}',\
```

Comment:
Just my 2 cents: I'm not sure this is a good solution, because this will log the current paranoia level (what user set up), not the rule's level.

Reply:
Oh, i just choose the wrong name for that variable, it wasn't suppose to be the one from `crs-setup.conf`. Thanks, i will fix it soon.